### PR TITLE
fix: add @types/webpack-env to apps that depend on it, fixes #8110

### LINKED ISSUE #8110: Typescript build fail when removing @types/storybook__react
My understanding is that the `@types` package is no longer needed for `@storybook/react`. However when i uninstall the module i get the below error when trying to build my project.

```
ERROR in [at-loader] ./node_modules/@storybook/react/dist/client/preview/index.d.ts:2:23 
    TS2688: Cannot find type definition file for 'webpack-env'.
```

Have i misunderstood ?

### PR DESCRIPTION
**Pull request by hipstersmoothie**
Issue: fixes  #8110

## What I did

add webpack-env types to react, they are needed for the build

## How to test

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by shilman** [comment]
@hipstersmoothie @kroeder Do we need to add these to all the other frameworks as well?

**Comment by ndelangen** [comment]
Yeah I think if we need to add this to 1 package, we need to add them to all that have typescript in them.

**Comment by hipstersmoothie** [comment]
Will do tomorrow!

**Comment by ndelangen** [comment]
Thank you! Really really appreciated!

**Comment by hipstersmoothie** [comment]
@ndelangen I added the types dep to all `app`s that had it in their tsconfig

**Comment by shilman** [comment]
@hipstersmoothie mind fixing the linting error?

> /tmp/storybook/app/preact/src/client/preview/render.tsx
  3:1  error  'common-tags' should be listed in the project's dependencies. Run 'npm i -S common-tags' to add it  import/no-extraneous-dependencies

File: app/preact/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@storybook/addons": "5.3.0-alpha.2",
     "@storybook/core": "5.3.0-alpha.2",
-    "common-tags": "^1.8.0",
```
Comment (ndelangen) [inline app/preact/package.json]: Did you intent to remove the dependency on `common-tags`

**Review by ndelangen — CHANGES_REQUESTED** [review]

File: app/preact/package.json (hunk 1 above)
Comment (hipstersmoothie) [reply app/preact/package.json]: whoops! all fixed now

**Review by ndelangen — APPROVED** [review]

**Pull request merged**